XenoHolographic-it/IAR-SDK-Sample-iOS
# IAR-4957 Marker, Hunt marker and Hunt Reward Details [Core]

# Overview

- Add ViewController to show marker details (also one for HuntMarkers and HuntRewards)

# Resources

Jira: https://iversoft.atlassian.net/browse/IAR-4957

# Technical Considerations

none

# Known Issues

none

# Testing Instructions

- For Markers: Open the OnDemand markers list, select one and see its details
- For HuntMarkers: Open the ARHunts list, select one and choose a marker to see its details
- For HuntRewards: Open the ARHunts list, select one and choose a reward to see its details
